components/Headers/ChapterHeader.js

```diff
@@ -0,0 +1,234 @@
+import { useState, useEffect } from "react";
```

Comment:
Great work 🚀 , but we can achieve this using `PageHeader` also as we need to hide 2 options for the chapter page only. @Gupta-Anubhav12 Any thoughts?